### ISSUE TITLE
Biospec - Dshbrd - Tubes Page Format for Clinical Sites

### ISSUE THREAD

**Issue by HopkinsDC** [episphere/connect#340]
We would like Labeling and Scanning pages to be tailored for each clinical site. Each clinical site has a slightly different set of tubes that they will collect and process, so the list of tubes displayed needs to be different for each site.

**Comment by brotzmanmj** [comment]
Preference would be to have a unique tube collection list for each clinical collection site. How onerous is this from a development perspective?

**Comment by wyattks** [comment]
Below is a link to a list with the tube types for each site.  If there is a "Y" in columns G - Q, the site will be using that tube type.  All research sites will be using the same tube types.

https://nih.app.box.com/file/728481198995?s=blsrnqult1gx8g8hg8zkmqf3zksg32ob

**Comment by wyattks** [comment]
Updated site mock-ups: https://nih.app.box.com/file/981712887106

**Comment by wyattks** [comment]
Putting on hold- just spoke with the sites and in interest for maximum flexibility with tubes sizes and amounts we are leaning towards 1 screen for all the sites.

**Comment by Davinkjohnson** [comment]
@wyattks Do we know which tubes they want on the "unified" screen? (I'll let @geniosys make the call on whether he wants a whole mock-up or just the listed items.)

**Comment by wyattks** [comment]
It's going to look similar to the NCI mock-up screen.   We have asked what their maximum numbers for each tube type would be.

**Comment by Davinkjohnson** [comment]
OK. Hendrick and I just chatted. and if the only change is that they'll be using a unified list of items (tubes, cups) then a list should suffice. but if there are other changes, we might need a mock-up. 

any idea on ETA for the list? (we had estimated several days of work for this, so if we're sticking to Dev testing the week of 8/8 we'll need the correct requirements pretty quick.)

**Comment by wyattks** [comment]
Waiting on KP sites to reply with tube information.  Once we have that information we will update the mock-up and reattach it here.
We are also adding back into the page the check boxes to indicate which tubes were collected.  We will NOT have the dropdown for reason not collected added back in.  The reason not collected information will not be known by the person entering the data into the dashboard.

**Comment by HopkinsDC** [comment]
Updated mock-up for clinical Collection Data Entry page (**confirmed by KP sites on 8/1**)


![Screen Shot 2022-08-01 at 12 54 15 PM](https://user-images.githubusercontent.com/85250133/182235029-a1dbc1fa-18d5-4ae1-b3c1-e379efabbc26.png)

**Comment by brotzmanmj** [comment]
We will need some new tube variables created in order to implement this. @HopkinsDC will create an issue specific to that request and assign it to Nicole and Madhuri.

**Comment by HopkinsDC** [comment]
After reviewing the data dictionary, we don't need any more tube variables created. The correct number of variables is already present
[Tube Variables_MasterSurveyComb_20220317.pdf](https://github.com/episphere/connect/files/9242178/Tube.Variables_MasterSurveyComb_20220317.pdf)
.

**Comment by wyattks** [comment]
Not creating for each site.
